lib/msal-node/src/utils/NodeConstants.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * http methods
+ */
+export enum method {
```

Comment:
Can this be `HttpMethod`, to be more specific?

Reply:
+1

Reply:
Done